scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1099,6 +1204,10 @@
                   (log/info "Registered with mesos with framework-id " framework-id)
                   (reset! fid framework-id)
                   (set-framework-id framework-id)
+                  (when (and gpu-enabled? (not (re-matches #"1\.\d+\.\d+" (:version master-info))))
```

Comment:
They only give the version as a string? If they had a major, minor, micro struct you could check if major is > 0